src/main.c

```diff
@@ -27,7 +27,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#ifdef _MSC_VER
+#include <processthreadsapi.h>
+#define localtime_r(timer, buf) localtime_s(buf, timer)
```

Comment:
Since the return type is also different, I think defining a proper helper function like in libjuice would be better:
```
static int get_localtime(const time_t *t, struct tm *buf) {
#ifdef _WIN32
        return localtime_s(buf, t) == 0 ? 0 : -1;
#else // POSIX
        return localtime_r(t, buf) != NULL ? 0 : -1;
#endif
}
```